### PR TITLE
Added r-mlrmbo recipe

### PR DESCRIPTION
**Pull request by khughitt**
Generated via the skeleton helper script.

Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dbast** [comment]
Requires r-smoof[version='>=1.5.1']

File: recipes/r-mlrmbo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    and is designed for both single- and multi- objective optimization with mixed continuous,
+    categorical and conditional parameters. The machine learning toolbox 'mlr' provide
+    dozens of regression learners to model the performance of the target algorithm with
+    respect to the parameter settings. It provides many different infill criteria to
+    guide the search process. Additional features include multi-point batch proposal,
+    parallel execution as well as visualization and sophisticated logging mechanisms,
+    which is especially useful for teaching and understanding of algorithm behavior.
+    'mlrMBO' is implemented in a modular fashion, such that single components can be
+    easily replaced or adapted by the user for specific use cases.
+  license_family: BSD
```
Comment (dbast) [inline recipes/r-mlrmbo/meta.yaml]:
```suggestion
  license_family: BSD
  license_file:
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/BSD_2_clause'
    - LICENSE
```

**Comment by khughitt** [comment]
> Requires r-smoof[version='>=1.5.1']

Good catch! I completely forgot to check to make sure all the dependencies are already packaged. Probably a good place to start next time heh.

It looks like smoof depends on one package ([mco](https://cran.r-project.org/web/packages/mco/index.html)) that needs to be added, but that should be it.

I'll work on recipes for mco & smoof and report back here once those are ready.

**Comment by dbast** [comment]
azure win build was fine. this most probably requires adding `provider: {win: azure}` to conda-forge.yml + rerender after feedstock conversion (as also done for r-smoof, r-mco).

**Pull request merged**